distro/scripts/startup.sh

```diff
@@ -15,6 +21,13 @@ Run the following commands from Windows or other WSL 2 distros to use.
     wsl.exe -d $WSL_DISTRO_NAME service wsl-vpnkit start
     wsl.exe -d $WSL_DISTRO_NAME service wsl-vpnkit stop
 
+or just create an alias:
+
+    echo 'alias vpnkit="wsl.exe -d wsl-vpnkit service wsl-vpnkit"' >/etc/profile.d/vpnkit.sh
+    source /etc/profile.d/vpnkit.sh
+    vpnkit start
+    vpnkit stop
```

Comment:
This script could vary between different distros, so I would rather omit this and leave the users to create their own setup.